brave/browser-laptop
# add right-click to delete ledger entry, add Bravery site settings page

Fix #2545

Test Plan:
1. browse to some sites and change the Brave settings on them
2. go to about:preferences#shields, and verify that the changed settings appear there
3. change the bravery global defaults and verify that the changed settings disappear

Fix #3829

Test plan:
1. go to about:preferences#security
2. right click on a site and select 'never include this site'
3. make sure the entry disappears from the table

Auditors: @bbondy @mrose17
